opencog/embodiment/ugly_male_soda-drinker_corpus.scm

```diff
@@ -1,360 +1,360 @@
+(setlocale LC_CTYPE "")
```

Comment:
I'm pretty sure that this will cause conflicts and bad behavior. You really want the locale to be utf8  The default locale should be fine, already.